profile/profile.md

```diff
@@ -0,0 +1,63 @@
+# Saiyeda Maham Shamail . Jarvis Consulting
+
+As a software developer I have been programming for the past 5 years, with one year in a professional setting. This has helped me develop exceptional problem solving skills. I have worked with technologies such as JavaScript, React.js, RDBMS (SQL), Redux and Python. I am passionate about learning new languages and technologies and I can do so very quickly.
```

Comment:
- please run a Grammarly spelling and grammar check for the entire profile
- it's a bit short. Try to write it 80-100 words. Talk about why you love new languages and tech. What's your strength? talk about your personality
- `I have worked with technologies such as JavaScript, React.js, RDBMS (SQL), Redux and Python.` remove this section as it's redundant to the Skills section.